### ISSUE TITLE
Node go from online to offline 

### ISSUE THREAD

**Issue by annie-incognito** [incognitochain/incognito-wallet#19]
Node from online to offline although it's online before and staked successfully

**Comment by peterincognito** [comment]
contacting @CodedInk @huglester @jgreene20 @iam_joshhamon for the IP that has "online-offline issue"

**Comment by peterincognito** [comment]
@iam_joshhamon: 34.68.82.141

**Comment by annie-incognito** [comment]
from: @CodedInk

description: 
I have multiple nodes and some are online and some are offline. All have been online constantly since mainnet launch and have been earning and fine. 

--
Just wanted to share I powered off and back on my nose that showed offline. It went back to offline and then went into “queueing” status now. 

---
So I just checked my app and I am showing "Offline" again on the same node. It is always the same node that shows offile? I wonder why the app would have an issue showing only one nodes status and that it wouldnt affect any nodes showing in the app?


from: @huglester
description: toda two of mine showed as offline too. but docker and all was ok

from: @iam_joshhamon
description: My virtual node shows offline in the app but the instance is running. Looks like between the last time I checked it and now I earned some PRV. I’ve tried restarting the app. Let it be or should I try something else?

from: @jgreene20
description: Now the node that was offline has queued and is back online and earning. But know the node that was online and earning has gone offline. They are just swapping offline and queuing status now
